angrmanagement/ui/widgets/qccode_edit.py

```diff
@@ -111,7 +111,7 @@ def get_context_menu(self):
             # decompiled function name in selection
             self._selected_node = under_cursor
             mnu.addActions(self.function_name_actions)
-            for entry in self.workspace.plugins.build_context_menu_function(self.workspace.instance.kb.functions[under_cursor.name]):
+            for entry in self.workspace.plugins.build_context_menu_functions(self.workspace.instance.kb.functions[under_cursor.name]):
```

Comment:
Should this be `([self.workspace...` instead of `(self.workspace...`? i.e. pass a list?

Reply:
@rhelmot yup, good catch